pkg/agent/route/route_linux_test.go

```diff
@@ -492,6 +493,318 @@ COMMIT
 -A ANTREA-POSTROUTING -m comment --comment "Antrea: masquerade LOCAL traffic" -o antrea-gw0 -m addrtype ! --src-type LOCAL --limit-iface-out -m addrtype --src-type LOCAL -j MASQUERADE --random-fully
 -A ANTREA-POSTROUTING -m comment --comment "Antrea: masquerade OVS virtual source IP" -s fc01::aabb:ccdd:eeff -j MASQUERADE
 COMMIT
+`, false, true)
+			},
+		},
+		{
+			name:                     "encap,wireguard,egress=true,multicastEnabled=true,proxyAll=true,nodeNetworkPolicy=true,nodeSNATRandomFully=true",
```

Comment:
would it make the tests a bit less verbose if we removed multicast and proxyAll?

Reply:
sure, I will update.

Reply:
I saw the change for the second test case but not this one, was it intentional?